### PR TITLE
Move multidict requirement to asyncio extras

### PR DESCRIPTION
**Pull request by johnlockwood-wf**
WIP until I can get the skynet tests to pass

Upgrade aiohttp. The latest aiohttp doesn't depend on multidict anymore, so our test that uses it doesn't need it anymore.
The multidict dep was also in the main requirements, but was only needed in the asyncio extras.

@Workiva/messaging-pp

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Raven
<details>
<summary>Number of Findings: 0</summary>

</details>

**Comment by brianshannan-wf** [comment]
This pr needs to be made against the develop branch

File: compiler/generator/python/generator.go [hunk 1]
```diff
@@ -1270,7 +1270,7 @@ func (g *Generator) generateProcessorFunction(method *parser.Method) string {
 		contents += tabtabtab + "with self._lock:\n"
 		contents += tabtabtabtab + fmt.Sprintf("_write_application_exception(ctx, oprot, \"%s\", ex_code=TApplicationExceptionType.INTERNAL_ERROR, message=e.message)\n", methodLower)
 	}
-	contents += tabtabtab + "raise\n"
+	contents += tabtabtab + "raise e\n"
```
Comment (aldenpeterson-wf) [inline compiler/generator/python/generator.go]: can you clarify why you are doing this?

This will lose the exception stack history.

**Comment by charliestrawn-wf** [comment]
@johnlockwood-wf did you mean to add the commits for switching to dep to this PR? The massive diff makes the original change impossible to review. Also we have a PR open for switching from godep to glide. I'm all for using dep but that was shot down as Workiva's dependency tracking tools don't support it yet.

**Comment by johnlockwood-wf** [comment]
@charliekump-wf I meant to make the crossrunner build following a change and it wouldn't build using the godep workspaces setup.

**Comment by johnlockwood-wf** [comment]
closing. will extract the cross test enhancements after the switch from godeps happens.

**Pull request closed without merging**